### PR TITLE
Add redirect for urls with uppercase chars, remove duplicate metrics [PREP-286]

### PR DESCRIPTION
**Pull request by binoculars**
## Purpose
Make provider names case insensitive (redirects to correct location if it is the correct provider name, but wrong case)

Also, get rid of duplicate trackEvent for sharing link buttons on the content page (Google Analytics).

## Ticket
https://openscience.atlassian.net/browse/PREP-286

### CODE REVIEW

File: app/controllers/content.js [hunk 1]
```diff
@@ -153,12 +153,6 @@ export default Ember.Controller.extend(Analytics, {
         shareLink(href, network, action, label) {
             const metrics = Ember.get(this, 'metrics');
 
-            metrics.trackEvent({
```
Comment (brianjgeiger) [inline app/controllers/content.js]: Is this related to this PR?

Comment (brianjgeiger) [inline app/controllers/content.js]: Why yes it is, right there in the description.

File: app/routes/provider.js [hunk 2]
```diff
@@ -10,8 +10,17 @@ export default Ember.Route.extend({
 
     beforeModel(transition) {
         const {slug} = transition.params.provider;
+        const slugLower = (slug || '').toLowerCase();
+
+        if (this.get('providerIds').includes(slugLower)) {
+            if (slugLower !== slug) {
+                const {pathname} = window.location;
+                window.location.pathname = pathname.replace(
+                    new RexExp(`^/preprints/${slug}`),
```
Comment (sloria) [inline app/routes/provider.js]: Typo here: `RegExp`
